### PR TITLE
[Improve][Connector-V2] Support INFINI Easysearch

### PR DESCRIPTION
**Pull request by luohoufu**
### Purpose of this pull request

Support INFINI Easysearch

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [x] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [x] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Review by hailin0 — COMMENTED** [review]
Please check ci errors

**Comment by luohoufu** [comment]
> Please check ci errors

Build Successful

File: docs/en/connector-v2/sink/Easysearch.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+# INFINI Easysearch
```
Comment (hailin0) [inline docs/en/connector-v2/sink/Easysearch.md:1]: Please refer to the new document style

https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/sink/IoTDB.md

File: docs/en/connector-v2/source/Easysearch.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
+# Easysearch
```
Comment (hailin0) [inline docs/en/connector-v2/source/Easysearch.md:1]: Please refer to the new document style

https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/MySQL-CDC.md

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-easysearch-e2e/src/test/resources/easysearch/easysearch_source_and_sink.conf [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    username = "admin"
+    password = "admin"
+    tls_verify_certificate = false
+    tls_verify_hostname = false
+
+    index = "st_index"
+    query = {"range": {"c_int": {"gte": 10, "lte": 20}}}
+    schema = {
+      fields {
+        c_map = "map<string, tinyint>"
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-easysearch-e2e/src/test/resources/easysearch/easysearch_source_and_sink.conf:42]: Please test read all datatypes of source

**Review by lightzhao — APPROVED** [review]
LGTM

**Review by hailin0 — DISMISSED** [review]

File: docs/en/connector-v2/sink/Easysearch.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+## Key features
+
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [x] [cdc](../../concept/connector-v2-features.md)
+
+:::tip
+
+Engine Supported
+
+* Supported all versions released by [INFINI Easysearch](https://www.infini.com/download/?product=easysearch).
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Easysearch.md:26]: image

Engine Supported lost.

File: docs/en/connector-v2/sink/Easysearch.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+:::
+
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Easysearch.md:29]: Data Mapping Lost. you can refer https://seatunnel.apache.org/docs/2.3.3/connector-v2/sink/Mysql

File: docs/en/connector-v2/source/Easysearch.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+
+> Easysearch source connector
+
+## Description
+
+Used to read data from INFINI Easysearch.
+
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/Easysearch.md:14]: Engine Support lost.
Please refer https://seatunnel.apache.org/docs/2.3.3/connector-v2/source/Mysql

File: docs/en/connector-v2/source/Easysearch.md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [stream](../../concept/connector-v2-features.md)
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [x] [column projection](../../concept/connector-v2-features.md)
+- [ ] [parallelism](../../concept/connector-v2-features.md)
+- [ ] [support user-defined split](../../concept/connector-v2-features.md)
+
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/Easysearch.md:27]: Data Type Mapping lost. https://seatunnel.apache.org/docs/2.3.3/connector-v2/source/Mysql

**Review by EricJoy2048 — DISMISSED** [review]

File: docs/en/connector-v2/sink/Easysearch.md (hunk 4 above)
Comment (luohoufu) [reply docs/en/connector-v2/sink/Easysearch.md:26]: updated

File: docs/en/connector-v2/sink/Easysearch.md (hunk 5 above)
Comment (luohoufu) [reply docs/en/connector-v2/sink/Easysearch.md:29]: updated

File: docs/en/connector-v2/source/Easysearch.md (hunk 6 above)
Comment (luohoufu) [reply docs/en/connector-v2/source/Easysearch.md:14]: updated

**Review by hailin0 — APPROVED** [review]

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**